utils/ort/src/main/kotlin/JavaBootstrapper.kt

```diff
@@ -131,7 +131,14 @@ object JavaBootstrapper {
             return Result.failure(it)
         }
 
-        val installDir = (ortToolsDirectory / "jdks" / pkg.distribution / pkg.distributionVersion)
+        return downloadJdk(pkg.links.pkgDownloadRedirect, pkg.distribution, pkg.distributionVersion)
+    }
+
+    /**
+     * Download the resolved JDK Url and return its directory on success, or an exception on failure.
+     */
+    fun downloadJdk(sdkUrl: String, distribution: String, distributionVersion: String): Result<File> {
+        val installDir = (ortToolsDirectory / "jdks" / distribution / distributionVersion)
```

Comment:
Related to my other comment about maybe not requiring the `javaVersion`, this function could be changed to straight take the `installDir` and let the caller set it.

Reply:
See above comment